### PR TITLE
set end-of-line to auto in prettier config

### PR DESCRIPTION
**Pull request by Lkleven**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
     "arrow-parens": [1, "as-needed"],
     "no-dupe-keys": 2,
     "no-dupe-args": 2,
-    "semi": 1
+    "semi": 1,
+    "end-of-line": "auto"
```
Comment (naivepop) [inline .eslintrc.json]: Burde denne ligge under "prettier/prettier"-regelen siden det er en prettier-option?

**Comment by Lkleven** [comment]
Moved configuration to correct place
Semantic version has been increased.  Not sure if it should be 5.3.0 rather than 5.2.1

File: .eslintrc.json [hunk 2]
```diff
@@ -20,7 +20,8 @@
     "prettier/prettier": [
       "warn",
       {
-        "singleQuote": true
+        "singleQuote": true,
+        "end-of-line": "auto"
```
Comment (asbjornh) [inline .eslintrc.json]: `endOfLine`

**Comment by asbjornh** [comment]
> Moved configuration to correct place
> Semantic version has been increased. Not sure if it should be 5.3.0 rather than 5.2.1

Yeah it's a little hard to know with this package... I think patch version increase is fine :)

File: .eslintrc.json (hunk 2 above)
Comment (Lkleven) [reply .eslintrc.json]: Sigh, fixed.

**Comment by asbjornh** [comment]
We should probably also upgrade this package in `@creuna/cli` once this version is publised to npm

**Pull request merged**